### PR TITLE
Vood 171 gif recording no longer working

### PR DESCRIPTION
**Pull request by PascalSchuermannOst**

### CODE REVIEW

File: src/node-modules-accessor/nodeModulesAccessor.ts [hunk 1]
```diff
@@ -14,19 +14,11 @@ export class NodeModulesAccessor {
...
-    [
-      NodeModulesKeys.ffmpegCoreJs,
-      {
-        sourcePath: ['node_modules', '@ffmpeg', 'core', 'dist'],
-        destinationPath: ['libs', '@ffmpeg', 'core', 'dist'],
-        fileName: 'ffmpeg-core.js',
+        sourcePath: ['node_modules', 'recordrtc'],
+        destinationPath: ['libs', 'recordrtc'],
+        fileName: 'RecordRTC.min.js',
         includeFolder: true,
```
Comment (GinoCardillo-OST) [inline src/node-modules-accessor/nodeModulesAccessor.ts:22]:
```suggestion
```
The min.js is enough.

**Comment by GinoCardillo-OST** [comment]
Recording the Gif is extremely slow but the results is good.
Recording the webm is faster but the results is worse.
Unfortunate...

**Review by alexandre-lagadec-ost — APPROVED** [review]

File: media/html/visjsDebuggerPanel.html [hunk 2]
```diff
@@ -237,40 +255,28 @@
         }
       }
 
-      function stopRecordingVisjs() {
+      function stopRecordingVisjsGif(render) {
```
Comment (alexandre-lagadec-ost) [inline media/html/visjsDebuggerPanel.html:258]:
```suggestion
      function stopRecordingVisjsGif() {
```

File: media/html/visjsDebuggerPanel.html [hunk 3]
```diff
@@ -237,40 +255,28 @@
...
+        if (vscode) {
+          recorder.stopRecording().then(function () {
+            recorder.getBlob().then(function (blob) {
+              downloadFile('export.gif', URL.createObjectURL(blob));
+            });
+          });
+        }
+      }
+
+      function stopRecordingVisjsWebm(render) {
```
Comment (alexandre-lagadec-ost) [inline media/html/visjsDebuggerPanel.html:268]:
```suggestion
      function stopRecordingVisjsWebm() {
```

**Review by alexandre-lagadec-ost — CHANGES_REQUESTED** [review]

**Review by alexandre-lagadec-ost — APPROVED** [review]

**Pull request closed without merging**